chsdi/templates/htmlpopup/gisgeol.mako

```diff
@@ -1,11 +1,31 @@
 <%inherit file="base.mako"/>
-<%!
+
+<%def name="table_body(c,lang)">
+<%
     import markupsafe
+    import glob,os
+    api_url = h.make_api_url(request, True)
+
     def br(text):
         return text.replace('\n', markupsafe.Markup('<br />'))
+
+    def hbytes(num):
+        for x in ['bytes','KB','MB','GB']:
+            if num < 1024.0:
+                return "%3.1f %s" % (num, x)
+            num /= 1024.0
+        return "%3.1f %s" % (num, 'TB')
+    
+    def findfiles(text):
+        result = []
+        for file in glob.glob('/var/local/cartoweb/downloads/gisgeol/%s*' % text ):
```

Comment:
I would remove `gisgeol`  and use only `/var/local/cartoweb/downloads/` as config variable. (in `developement.ini.in`)
and then a folder which name is the `layer name`. This way we can also remove the Alias. So the path would be `/var/local/cartoweb/downloads/ch.swisstopo.geologie-gisgeol/`.